.github/workflows/build-test-deploy.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           map: |
             {
               "main": {
-                "BUILD_ALIAS": "prod",
+                "BUILD_ALIAS": "production",
```

Comment:
Ahh... I see! The Firestore-backup storage locations are hardcoded against `production`. Probably that should be configuration not in code!!

Currently I think we don't have any config for `admin`, so doing it properly isn't completely trivial. If we were to do it right we should also consider config keys that are used by multiple modules. Interestingly, the functions region is such a case; currently it's manually duplicated to both `functions` & `hosting`.

Maybe the end goal is a general purpose config transformer:
- In: firebase config, firebase service account, custom config
- Out: functions config, hosting config, admin config, common config?!, github config

I'd like:
- `yarn config:init` => replacing bootstrap; generate all local configs (functions, hosting, admin, common)
- `yarn config:deploy` => push config to github

...I should put this in a PBI...